server/ai_http.go

```diff
@@ -96,26 +97,95 @@ func (h *lphttp) StartLiveVideoToVideo() http.Handler {
 
 		remoteAddr := getRemoteAddr(r)
 		ctx := clog.AddVal(r.Context(), clog.ClientIP, remoteAddr)
+		requestID := string(core.RandomManifestID())
 
-		// skipping handleAIRequest for now until we have payments
-
-		var (
-			mid        = string(core.RandomManifestID())
-			pubUrl     = TrickleHTTPPath + mid
-			subUrl     = pubUrl + "-out"
-			controlUrl = pubUrl + "-control"
-		)
-		jsonData, err := json.Marshal(
-			&worker.LiveVideoToVideoResponse{
-				PublishUrl:   pubUrl,
-				SubscribeUrl: subUrl,
-				ControlUrl:   controlUrl,
-			})
+		payment, err := getPayment(r.Header.Get(paymentHeader))
```

Comment:
we aren't dealing with payments at this point so lets omit this and @leszko can handle that in https://github.com/livepeer/go-livepeer/pull/3261

Reply:
This is being used for logging the eth address of the gateway downstream, is there another way we can get it?

Reply:
Used an empty ETH address for now

Reply:
yes, I'll handle this in #3261